src/topic_monitor.cpp

```diff
@@ -298,6 +303,65 @@ void TopicMonitor::unpause()
 }
 
 
+#if OPENDDS_MAJOR_VERSION == 3 && OPENDDS_MINOR_VERSION >= 24
```

Comment:
Do we need these guards? We haven't officially released a version for opendds-monitor... I assume that once we're ready to do that we can just make it a requirement for it to be compiled with the minimum appropriate version of OpenDDS. Or are we trying to support any version after the release of DynamicData?

Reply:
Based on the CI setup, my goal was to have this monitor revision compile with both the latest release of OpenDDS and the current master branch.  It doesn't have to be that way, but it's not difficult to do and it's easy to determine when the conditionals can be dropped based on the OpenDDS release version.